index.php

```diff
@@ -52,6 +230,55 @@
     2) For a given patient, all suitable matches ranked from worst to best from all the registered hospitals are displayed.</b><br> 
   </h3>
 </div>
+<footer class="footer-distributed">
+
+<div class="footer-left">
+    <img src="images/logo3.png">
+  <h3>About<span>Eduonix</span></h3>
+
+  <p class="footer-links">
+    <a href="#">Home</a>
+    |
+    <a href="#">Blog</a>
+    |
+    <a href="#">About</a>
+    |
+    <a href="#">Contact</a>
+  </p>
+
+  <p class="footer-company-name">© 2019 Eduonix Learning Solutions Pvt. Ltd.</p>
+</div>
+
+<div class="footer-center">
+  <div>
+    <i class="fa fa-map-marker"></i>
+      <p><span>309 - Rupa Solitaire,
+       Bldg. No. A - 1, Sector - 1</span>
+      Mahape, Navi Mumbai - 400710</p>
+  </div>
+
+  <div>
+    <i class="fa fa-phone"></i>
+    <p>+91 22-27782183</p>
+  </div>
+  <div>
+    <i class="fa fa-envelope"></i>
+    <p><a href="mailto:support@eduonix.com">support@eduonix.com</a></p>
+  </div>
+</div>
+<div class="footer-right">
+  <p class="footer-company-about">
+    <span>About the company</span>
+    We offer training and skill building courses across Technology, Design, Management, Science and Humanities.</p>
```

Comment:
This is not what this project does...lol
Do not forget to update after copy-pasting the code :)

Reply:
what upadate u want